packages/kbn-management/settings/utilities/category/const.ts

```diff
@@ -39,4 +40,5 @@ export const CATEGORY_ORDER = [
   SECURITY_SOLUTION_CATEGORY,
   TIMELION_CATEGORY,
   VISUALIZATION_CATEGORY,
+  DEV_TOOLS_CATEGORY,
```

Comment:
@serenachou  @julianrosado I wasn't sure where to place this in the order so I just added it to the end, let me know if you have a different preference.

Reply:
I don't feel strongly about it. The goal was to make it available and convenient, not necessarily highly (above the fold) visible?